synthesizer/process/src/tests/test_process.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
```

Comment:
This license needs to be updated, see e.g. https://github.com/AleoNet/snarkVM/blob/mainnet-staging/fields/src/fp12_2over3over2.rs#L1